### PR TITLE
FAQ - Make it more clear which QR code type an entry refers to

### PR DESCRIPTION
**Pull request by heinezen**
Fixes https://github.com/corona-warn-app/cwa-website/issues/1172

New terminology:

**English**
* PCR-Test QR code
* Rapid-Test QR code
* Event QR code

**German**
* PCR-Test-QR-Code
* Schnelltest-QR-Code
* Event-QR-Code

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -240,7 +240,7 @@
...
-                            "In summary, the proposed solution allows a host to create a venue through CWA. All necessary data about the venue is encoded in a QR code which can be presented on a mobile device or printed out, for example to be posted at the entrance of the venue. An attendee can check in to the venue by scanning the QR code. Check-ins are stored locally on the mobile device and deleted automatically after two weeks.",
+                            "In summary, the proposed solution allows a host to create a venue through CWA. All necessary data about the venue is encoded in an (Event) QR code which can be presented on a mobile device or printed out, for example to be posted at the entrance of the venue. An attendee can check in to the venue by scanning the QR code. Check-ins are stored locally on the mobile device and deleted automatically after two weeks.",
```
Comment (DerVogel2020) [inline src/data/faq.json:243]: Why is Event here in Braces?

File: src/data/faq_de.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
                         "anchor": "rat_how_get_result",
                         "active": true,
                         "textblock": [
-                            "Über den Bereich 'Test registrieren' auf dem Startbildschirm Ihrer Corona-Warn-App, können Sie einen Test hinzufügen, indem Sie den QR-Code scannen. Um das Ergebnis möglichst zeitnah in der App zu erhalten, können Sie den Test in der App aktualisieren, bis das Ergebnis vorliegt. Zwar ruft die Corona-Warn-App Testergebnisse auch automatisch im Hintergrund ab, je nach Betriebssystem kann das aber etwas länger dauern."
+                            "Über den Bereich 'Test registrieren' auf dem Startbildschirm Ihrer Corona-Warn-App, können Sie einen Test hinzufügen, indem Sie den Schnelltest-QR-Code scannen. Um das Ergebnis möglichst zeitnah in der App zu erhalten, können Sie den Test in der App aktualisieren, bis das Ergebnis vorliegt. Zwar ruft die Corona-Warn-App Testergebnisse auch automatisch im Hintergrund ab, je nach Betriebssystem kann das aber etwas länger dauern."
```
Comment (DerVogel2020) [inline src/data/faq_de.json:110]: Um einen Test zu registrieren, können doch sowohl PCR-Test-QR-Codes als auch Schnelltest-QR-Codes über den Bereich "Test registrieren" gescannt werden.

File: src/data/faq_de.json [hunk 3]
```diff
@@ -241,7 +241,7 @@
...
+                            "Zusammenfassend lässt sich sagen: Die vorgeschlagene Lösung ermöglicht es einer Gastgeberin, einem Gastgeber, einen Veranstaltungsort über die CWA zu erstellen. Alle notwendigen Daten über den Veranstaltungsort sind in einem (Event-)QR-Code eingebettet, der entweder auf einem mobilen Gerät präsentiert oder aber auch ausgedruckt werden kann, beispielsweise am Eingang des Veranstaltungsortes. Veranstaltungsteilnehmende können am Veranstaltungsort einchecken, indem sie den QR-Code scannen. Check-ins werden lokal auf dem mobilen Gerät gespeichert und nach zwei Wochen automatisch gelöscht.",
```
Comment (DerVogel2020) [inline src/data/faq_de.json:244]: Same as above.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json:243]: Because the data is encoded in an QR code and not into an Event-QR code.

File: src/data/faq_de.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq_de.json:110]: This entry is specifically about rapid tests, so I included the more detailed wording.

**Comment by MikeMcC399** [comment]
This PR solves some issues mentioned in https://github.com/corona-warn-app/cwa-website/issues/1238.

**Pull request merged**